### ISSUE TITLE
Better support for unescaped JSON objects

### ISSUE THREAD

**Issue by lpzantony** [nwrkbiz/Cpp-SupportLibrary#5]
`Objet.toUnescapedString()` does not fully use unescaped versions of the `Object`.

I would like to offer a branch fixing this.

**Comment by nwrkbiz** [comment]
Feel free to provide an pull request :)

**Comment by lpzantony** [comment]
Yes I have been trying yesterday but the github's UI won't let me create one. It apparently needs a branch, but if I try to push a branch I get rejected. I also tried to create a branch for this very issue but same problem, the github's UI won't let me :(

**Comment by nwrkbiz** [comment]
Can you describe your changes?

**Comment by lpzantony** [comment]
Attached is the diff from my commit:
[patch.txt](https://github.com/nwrkbiz/Cpp-SupportLibrary/files/11365895/patch.txt)
